app/src/main/java/uk/ac/ebi/atlas/search/metadata/CellTypeWheelService.java

```diff
@@ -13,19 +15,32 @@
 @Service
 public class CellTypeWheelService {
     private final CellTypeWheelDao cellTypeWheelDao;
+    private final FeaturedSpeciesService featuredSpeciesService;
 
-    public CellTypeWheelService(CellTypeWheelDao cellTypeWheelDao) {
+    public CellTypeWheelService(CellTypeWheelDao cellTypeWheelDao,
+                                FeaturedSpeciesService featuredSpeciesService) {
         this.cellTypeWheelDao = cellTypeWheelDao;
+        this.featuredSpeciesService = featuredSpeciesService;
     }
 
     public ImmutableSet<ImmutablePair<ImmutableList<String>, String>> search(String searchTerm, String species) {
-        return cellTypeWheelDao.facetSearchCtwFields(searchTerm, species)
+        ImmutableList<String> allSpeciesNames = featuredSpeciesService.getSpeciesNamesSortedByExperimentCount();
+        var isSpeciesSearch = allSpeciesNames.contains(StringUtils.capitalize(searchTerm));
+
+        return isSpeciesSearch ? cellTypeWheelDao.speciesSearchCtwFields(searchTerm, species)
                 .stream()
                 // This will effectively “explode” tuples and aggregate experiment accessions (the last element in the
                 // tuple) to the organisms, organism parts and cell types
                 .map(this::addTailToEveryHeadSublist)
                 .flatMap(ImmutableList::stream)
-                .collect(toImmutableSet());
+                .collect(toImmutableSet()) :
+                cellTypeWheelDao.facetSearchCtwFields(searchTerm, species)
+                        .stream()
```

Comment:
There is code duplication here. You can extract the results from the DAO to a variable and do the streaming on that.